core/insertion_marker_manager.ts

```diff
@@ -460,150 +447,135 @@ export class InsertionMarkerManager {
   /**
    * Show an insertion marker or replacement highlighting during a drag, if
    * needed.
-   * At the beginning of this function, this.localConnection_ and
-   * this.closestConnection_ should both be null.
+   * At the beginning of this function, this.localConnection and
+   * this.closestConnection should both be null.
    *
-   * @param candidate An object containing a local connection, a closest
-   *     connection, and a radius.
+   * @param newCandidate A new candidate connection that may replace the current
+   *     best candidate.
    */
-  private maybeShowPreview_(candidate: CandidateConnection) {
+  private maybeShowPreview(newCandidate: CandidateConnection|null) {
     // Nope, don't add a marker.
-    if (this.wouldDeleteBlock_) {
+    if (this.wouldDeleteBlockInternal) {
       return;
     }
-    const closest = candidate.closest;
-    const local = candidate.local;
 
     // Nothing to connect to.
-    if (!closest) {
+    if (!newCandidate) {
       return;
     }
 
+    const closest = newCandidate.closest;
+
     // Something went wrong and we're trying to connect to an invalid
     // connection.
-    if (closest === this.closestConnection_ ||
+    if (closest === this.activeCandidate?.closest ||
```

Comment:
I preserved this because I'm trying to keep the behaviour exactly the same, but I *don't know* what this is checking or why it's important.